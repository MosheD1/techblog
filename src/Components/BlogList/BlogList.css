.center {
    margin: auto;
    width: 66%;
}