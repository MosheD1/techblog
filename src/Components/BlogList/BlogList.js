import React from 'react';
import BlogCard from '../BlogCard/BlogCard';

import './BlogList.css';

const BlogList = ({blogs}) => {
    return (
        <div className='center'>
            {
                blogs.map((b, i) => {
                    return (<BlogCard key={i} title={b.title} />);
                })
            }
        </div>
    )
}

export default BlogList;