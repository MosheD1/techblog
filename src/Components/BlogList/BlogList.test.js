import React from 'react';
import BlogList from './BlogList';
import {shallow, render, mount} from 'enzyme';

describe('BlogList component', () => {
    it('expect BlogCard to render', () => {
        const mockBlogs = [
            {
                title: 'The new world'
              },
              {
                title: 'The new dicovery'
              }
        ]

        expect(shallow(<BlogList blogs={mockBlogs}/>)).toMatchSnapshot();
    });
})