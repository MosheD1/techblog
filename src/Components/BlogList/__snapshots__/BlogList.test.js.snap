// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogList component expect BlogCard to render 1`] = `ShallowWrapper {}`;
